### ISSUE TITLE
Error after update to 4.0.11

### ISSUE THREAD

**Issue by sajjanstha** [FosterCommerce/klaviyoconnect#89]
After I ran update for the plugin from 4.0.4 to 4.0.11, it throws error as shown below :
```
Performing update with Composer ... done
Applying new migrations ... error: The command "'/app/craft' 'migrate/all' '--no-backup' '--no-content'" failed.
Exit Code: 255(Unknown error)
Working directory: /app
Output:
================

Error Output:
================

Output:
```
When I check the phperrors.log file, I see these errors:
```
PHP Fatal error:  Type of fostercommerce\klaviyoconnect\Plugin::$hasCpSettings must not be defined (as in class craft\base\Plugin) in /app/vendor/fostercommerce/klaviyoconnect/src/Plugin.php on line 130
PHP Warning:  Function must be enabled in php.ini by setting 'xdebug.mode' to 'develop' in /app/vendor/yiisoft/yii2/base/ErrorException.php on line 50
PHP Fatal error:  Uncaught yii\base\ErrorException: Function must be enabled in php.ini by setting 'xdebug.mode' to 'develop' in /app/vendor/yiisoft/yii2/base/ErrorException.php:50
Stack trace:
#0 {main}
  thrown in /app/vendor/yiisoft/yii2/base/ErrorException.php on line 50
```

**Comment by fantasticmachine** [comment]
Hi @sajjanstha 
I've just released 4.0.12 which fixes this. Again, apologies for the issues here. I think it's now stable once again.
Pete

**Comment by sajjanstha** [comment]
Hi @fantasticmachine, new update fixed this issue but I am facing new issue as mentioned [here](https://github.com/FosterCommerce/klaviyoconnect/issues/90)
